sycl/doc/extensions/proposed/sycl_ext_oneapi_work_group_static.asciidoc

```diff
@@ -127,11 +128,7 @@ private:
 } // namespace sycl::ext::oneapi::experimental
```

Comment:
The synopsis above no longer matches the definitions you have below.  Can you update the synopsis?